templates/v2/professional/_professionals_list.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends 'v2/users_list/_subjects_list.html.twig' %}
+
+{% block listRows %}
+    {% for professional in professionals %}
+        {{ include('v2/professional/_professional_list_row.html.twig', {professional}) }}
+    {% endfor %}
+{% endblock %}
+
+{% block pagination %}
+    {{ knp_pagination_render(professionals, 'v2/vault/components/_pagination.html.twig', {}, {
```

Comment:
I feel this template is exactly the same as _benef_list.html.twig, with `beneficiaries` replaced with `professionals`, if you make the list_row mor common

Reply:
I'm not sure we can make a common list_row template as they are quite different. If we do this, i'm not sure how to condition the rendering

Reply:
We could make a twig extension such as "getUserCard()" (but it's not very accurate because we provide beneficiairies or pros), or "getSubjectCard()", but a subject is not really an entity...